monitoring.yaml

```diff
@@ -25,6 +25,9 @@ components:
   - repo_name: "tiflow"
     monitor_path: "dm/metrics/grafana"
     rule_path: "dm/metrics/alertmanager"
+  - repo_name: "migration"
```

Comment:
what is migrations?

Reply:
https://github.com/tikv/migration
tools for tikv